### DISCUSSION TITLE
Enforce Conventional Commits in PRs

### DISCUSSION THREAD

**Discussion by sschuberth** [jreleaser/jreleaser#1130] [Q&A]
IIUC, JReleaser relies on Conventional Commits to create release notes, but there's no way to use JReleaser to enforce Conventional Commits in PRs *before* any release happens. Is there any recommendation from the JReleaser community which tool / GitHub Action to use for this? Like maybe https://github.com/conventional-changelog/commitlint?

**Comment by aalmiray** [comment]
JBang uses https://github.com/Ezard/semantic-prs
We're thinking in using the same for JReleaser itself.
